### PR TITLE
Display rewards as they accumulate

### PR DESCRIPTION
**Pull request by r-czajkowski**
Closes: #1504

Currently, the dapp displays only those rewards that are available for withdrawal. In this PR the app displays rewards accumulating and clearly indicates which ones of them are available for withdrawal. Also, I implemented UI according to view from Figma except for new fonts and rewards in $.

Additional the PR, adds a `Datatable` component and replace an HTML list with a table.

obraz

obraz

obraz

### CODE REVIEW

File: solidity/scripts/rewards.js [hunk 1]
```diff
@@ -40,74 +40,87 @@ function formatAmount(amount, decimals) {
...
-  const requestor = accounts[5];
-  const owner = accounts[0];
+  try {
+
+    const accounts = await getAccounts();
+    const token = await KeepToken.deployed();
+    const tokenStaking = await TokenStaking.deployed();
+    const contractService = await KeepRandomBeaconService.deployed();
+    const keepRandomBeaconService = await KeepRandomBeaconServiceImpl.at(contractService.address);
+    const keepRandomBeaconOerator = await Operator.deployed();
```
Comment (lukasz-zimnoch) [inline solidity/scripts/rewards.js]: Typo `keepRandomBeaconOerator`. Also, please rename `Operator` to the full contract name.

File: solidity/scripts/rewards.js [hunk 2]
```diff
@@ -40,74 +40,87 @@ function formatAmount(amount, decimals) {
...
+    const entryFeeEstimate = await keepRandomBeaconService.entryFeeEstimate(0)
+    await keepRandomBeaconService.methods['requestRelayEntry()']({value: entryFeeEstimate, from: requestor})
+  
+    await registerNewGroups(10);
+  
+    const numberOfGroups = await keepRandomBeaconOerator.numberOfGroups()
+    const firstActiveIndex = await keepRandomBeaconOerator.getFirstActiveGroupIndex()
+  
+    for (let i = 0; i < firstActiveIndex; i++) {
+      const groupPubKey =  await keepRandomBeaconOerator.getGroupPublicKey(i);
```
Comment (lukasz-zimnoch) [inline solidity/scripts/rewards.js]: Please avoid abbreviations `groupPubKey`

File: solidity/scripts/rewards.js [hunk 3]
```diff
@@ -40,74 +40,87 @@ function formatAmount(amount, decimals) {
...
+    await keepRandomBeaconOerator.emitRewardsWithdrawnEvent(accounts[1], 3)
+    await keepRandomBeaconOerator.emitRewardsWithdrawnEvent(accounts[1], 1)
+    await keepRandomBeaconOerator.emitRewardsWithdrawnEvent(accounts[1], 5)
+    await keepRandomBeaconOerator.emitRewardsWithdrawnEvent(accounts[1], 6)  
+  
+    async function registerNewGroups (numberOfGroups) {
+      const groupReward = web3.utils.toWei('14500', 'Gwei');
+      for (let i = 0; i < numberOfGroups; i++) {
+        console.log('register group', i+1);
+        const groupPubKey = crypto.randomBytes(32);
```
Comment (lukasz-zimnoch) [inline solidity/scripts/rewards.js]: Same here regarding abbreviations.

**Comment by pdyraga** [comment]
Can we drop some screenshot to the PR description?

**Pull request merged**